fjord/base/management/commands/celeryd.py

```diff
@@ -0,0 +1,89 @@
+"""
+
+Start the celery daemon from the Django management command.
+
+"""
+from __future__ import absolute_import, unicode_literals
+
+import os
+import sys
+
+from django.core.management.base import BaseCommand
+
+import celery
+from celery.bin import worker
+
+from fjord.celery import app
+
+
+worker = worker.worker(app=app)
```

Comment:
`worker` is a module, but then this line changes it into something else. Is there a better way to do this that doesn't involve overloading "worker"?